cmd/influxd/server_integration_test.go

```diff
@@ -837,7 +850,7 @@ func TestSingleServer(t *testing.T) {
 
 	nodes := createCombinedNodeCluster(t, testName, dir, 1, 8090, nil)
 
-	runTestsData(t, testName, nodes, "mydb", "myrp")
+	runTestsData(t, testName, nodes, "mydb", "myrp", 7)
```

Comment:
You don't mean to check this in, I'm sure. But I'm glad you found it useful. :-)